packages/react-mutation-mapper/src/store/DefaultMutationMapperStore.ts

```diff
@@ -106,6 +106,7 @@ class DefaultMutationMapperStore<T extends Mutation>
 
     @observable
     private _selectedTranscript: string | undefined = undefined;
+    sampleDataByCodon?: any[] = [];
```

Comment:
This should be a `computed` value. Also should be a well defined type, not just `any`. We may need to do this in a more specific store, like `StudyViewMutationMapperStore`.

Reply:
The type is updated in the new implementation.